src/common/reader.js

```diff
@@ -1178,10 +1179,11 @@ class Reader {
 					this._updateState({ selectedAnnotationIDs: ids });
 
 					// Don't navigate to annotation or focus comment if opening a context menu
+					// unless it is a note that was just created via shortcut
 					if (!triggeringEvent || triggeringEvent.button !== 2) {
 						if (triggeredFromView) {
 							if (['note', 'highlight', 'underline'].includes(annotation.type)
-								&& !annotation.comment && (!triggeringEvent || !('key' in triggeringEvent))) {
+								&& !annotation.comment && (!triggeringEvent || !('key' in triggeringEvent) || (triggeringEvent.code.includes("Digit") && annotation.type === "note"))) {
```

Comment:
We can probably just remove `!('key' in triggeringEvent)` instead of checking for digit, as it was added to prevent focusing on note annotation comment when it was created from keyboard events, as we discussed in another thread.

Reply:
Right, so I then just left the `annotation.type === "note"` part, as we always want to let the user type once they created a note via shortcut. It does do exactly what I advocated against with other annotation type, when selecting it with Enter moves focus away into the sidebar. But I feel like in this case, it is not unreasonable, since the only thing you would want to do with an empty note is edit it.